### ISSUE TITLE
Posibility to customize with themes the "actions panel".

### ISSUE THREAD

**Issue by ludg1e** [mmatyas/pegasus-frontend#790]
Add the posibility to customize this:
![image](https://user-images.githubusercontent.com/77110723/107151416-c2bfe700-6962-11eb-976a-63e419a7ddc5.png)

**Comment by mmatyas** [comment]
Hi, this panel is not part of the themes, but Pegasus' main interface, which is independent from them. You can, however, create as many menus as you wish in the themes.

**Comment by ludg1e** [comment]
Yes, but if you create a theme, then for example when you want to quit or open the options of Pegasus, the menu looks like it's out of context of all the theme, that's why I'm suggesting this.
